tests/snuba/api/endpoints/test_organization_trace_details.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import absolute_import
+
+import pytz
+from datetime import datetime, timedelta
+
+from django.core.urlresolvers import reverse
+from django.utils import timezone
+from mock import patch
+
+from sentry.testutils import APITestCase, SnubaTestCase
+from sentry.utils.samples import load_data
+
+
+class GetOrganizationTraceDetailsTest(APITestCase, SnubaTestCase):
+    endpoint = 'sentry-api-0-organization-trace-details'
+    trace_id = 'a7d67cf796774551a95be6543cacd459'
+
+    def setUp(self):
+        super(GetOrganizationTraceDetailsTest, self).setUp()
+        self.url = reverse(self.endpoint, args=[self.organization.slug, self.trace_id])
```

Comment:
[Nit] Since you defined endpoint, you can just call `self.get_response(slug, trace_id)`. You only need `endpoint` if you want to use `get_response`, otherwise no need to define.